chore(docs): update version policy to remove Optimize preview releases (#1544)
camunda/camunda-platform-docs: optimize/self-managed/optimize-deployment/version-policy.md

## Release Cadence

> Camunda Optimize will get new releases every month being at least an alpha release and/or a patch release. Every six months, a new minor/major will be published. The release dates are be aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).

Reviewer: ```suggestion
Camunda Optimize will get a new release every month. Every six months, a new minor/major will be published, all other months it will be an alpha and/or patch release. The release dates are aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).
```
Or sth else for the first sentence, phrasing just felt off
